gpu_bdb/queries/q19/gpu_bdb_query_19_dask_sql.py

```diff
@@ -91,12 +91,14 @@ def main(data_dir, client, c, config):
 
     # second step -- Sentiment Word Extraction
     merged_df["pr_review_sk"] = merged_df["pr_review_sk"].astype("int32")
+    
     merged_df["pr_review_content"] = merged_df.pr_review_content.str.lower()
+    
     merged_df["pr_review_content"] = merged_df.pr_review_content.str.replace(
-        [".", "?", "!"], [eol_char], regex=False
+        "|".join(["\. ", "\? ", "! "]), eol_char, regex=True
     )
```

Comment:
Lets not use regex because that has perf implications.  I would suggest chaining operations instead.  Something like: 

```python
for char in [".", "?", "!"]:
  merged_df["pr_review_content"]  = merged_df.pr_review_content.str.replace(char, eol_char, regex=False) 
```